R/adverb-safely.R

```diff
@@ -10,7 +10,11 @@
 #' function (a verb). If you'd like to include a function created an adverb
 #' in a package, be sure to read [faq-adverbs-export].
 #'
-#' @inheritParams map
+#' @param .f A function to modify, specified in one of the following ways:
+#'   * A named function, e.g. `mean`.
+#'   * An anonymous function, e.g. `function(x) x + 1`.
+#'   * A lambda function, e.g. `\(a) a + 1`.
+#'   * A formula, e.g. `~ .x + 1`. Only appropriate for simple tasks.
```

Comment:
What do you mean by simple tasks here?

Reply:
I'm trying to imply that you shouldn't expect `...` to work as in #812.